[Python] Added dramatiq integration getting started (#11015)
getsentry/sentry-docs: docs/platforms/python/integrations/dramatiq/index.mdx

```

## Configure

Add `DramatiqIntegration()` to your `integrations` list:

<SignInNote />

In addition to capturing errors, you can monitor interactions between multiple services or applications by [enabling tracing](/concepts/key-terms/tracing/). You can also collect and analyze performance profiles from real users with [profiling](/product/explore/profiling/).

Select which Sentry features you'd like to install in addition to Error Monitoring to get the corresponding installation and configuration instructions below.

<OnboardingOptionButtons
  options={[
    'error-monitoring',
    'performance',
    'profiling',
  ]}
/>

```python {"onboardingOptions": {"performance": "6-8", "profiling": "9-12"}}
import sentry_sdk
from sentry_sdk.integrations.dramatiq import DramatiqIntegration

sentry_sdk.init(
    dsn="___PUBLIC_DSN___",
    # Set traces_sample_rate to 1.0 to capture 100%
    # of transactions for tracing.
    traces_sample_rate=1.0,
    # Set profiles_sample_rate to 1.0 to profile 100%
    # of sampled transactions.
    # We recommend adjusting this value in production.
    profiles_sample_rate=1.0,
    integrations=[
        DramatiqIntegration(),
    ],
)
```

## Verify

> 

Reviewer: ```suggestion

Trigger an error in your code to verify that the integration is sending events to Sentry.

```

```python
import dramatiq

import sentry_sdk
sentry_sdk.init(...)  # same as above

@dramatiq.actor(max_retries=0)
def dummy_actor(x, y):
    return x / y

dummy_actor.send(5, 0)

```

> Running this will create an error event (`ZeroDivisionError`) that will be sent to [sentry.io](https://sentry.io).

Reviewer: ```suggestion
Running this will create an error event (`ZeroDivisionError`) that you should be able to see in [sentry.io](https://sentry.io).
```

Select which Sentry features you'd like to install in addition to Error Monitoring to get the corresponding installation and configuration instructions below.

<OnboardingOptionButtons
  options={["error-monitoring", "performance", "profiling"]}
/>

```python {"onboardingOptions": {"performance": "6-8", "profiling": "9-12"}}
import sentry_sdk
from sentry_sdk.integrations.dramatiq import DramatiqIntegration

sentry_sdk.init(
    dsn="___PUBLIC_DSN___",
    # Set traces_sample_rate to 1.0 to capture 100%
    # of transactions for tracing.
    traces_sample_rate=1.0,
    # Set profiles_sample_rate to 1.0 to profile 100%
    # of sampled transactions.
    # We recommend adjusting this value in production.
    profiles_sample_rate=1.0,
    integrations=[
        DramatiqIntegration(),
    ],
)
```

## Verify

Trigger an error in your code to verify that the integration is sending events to Sentry.

```python
import dramatiq

import sentry_sdk
sentry_sdk.init(...)  # same as above

@dramatiq.actor(max_retries=0)
def dummy_actor(x, y):
    return x / y

dummy_actor.send(5, 0)

> 

Reviewer 2: ```suggestion
```
